schemas/test/en/test_currency.json

```diff
@@ -9,7 +9,6 @@
     "description": "A questionnaire to test currency input type",
     "messages": {
```

Comment:
More of a question to other devs. Should I remove the NUMBER_TOO_LARGE message since it is generic and we have more dynamic error handling.